app/controllers/frontend_controller.rb

```diff
@@ -164,7 +164,7 @@ def ensure_master_domain
   end
 
   def error_request_data
-    defined?(Airbrake) ? airbrake_request_data : {}
+    {}
   end
```

Comment:
Imo we should remove this method completely now.
![image](https://user-images.githubusercontent.com/11318903/69792764-83770180-11c7-11ea-8605-71ddace53142.png)
![image](https://user-images.githubusercontent.com/11318903/69792802-925db400-11c7-11ea-91cb-233010f789ea.png)

We should get rid of that `options` param and simplify the method `notify_about_wrong_domain` to:

```ruby
def notify_about_wrong_domain(url, expected)
   # if there is no referer there is not a link leading there and we don't care about it
   return unless request.referer
   Rails.logger.info("ThreeScale::Warnings::WrongDomain: URL '#{url}' is #{expected} only.")
end
```

In fact, we may be able to completely remove that method as well and then turn `render_wrong_domain_error` into:

```ruby
def render_wrong_domain_error(expected)
  Rails.logger.info("ThreeScale::Warnings::WrongDomain: URL '#{request.url}' is #{expected} only.") if request.referer
  render_error "The path '#{request.path}' is not accessible on domain #{request.host}", status: :not_found
end
```

Reply:
Is this useless?
```ruby
      options[:error_message] = "URL '#{url}' is #{expected} only."
      options[:error_class] = 'ThreeScale::Warnings::WrongDomain'
```

Reply:
yes @josemigallas , because the only places where this method is called, later they only call `render_wrong_domain_error` which does not use that at all :joy: 

This was useful only for Airbrake really long ago, and they forgot to remove those options [when they removed the Airbrake line](https://github.com/3scale/system/commit/c6361dfe85a4a70e37fa50de9103969762a0c4b7#diff-618ac6be5874259ac2ff1b1e897fb693L32).